src/AlertsManagement/AlertsManagement/AlertCommands/GetAzureAlertHistory.cs

```diff
@@ -0,0 +1,73 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Rest.Azure;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.AlertsManagement.OutputModels;
+using Microsoft.Azure.Management.AlertsManagement.Models;
+
+namespace Microsoft.Azure.Commands.AlertsManagement
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "AlertHistory", DefaultParameterSetName = ByIdParameterSet)]
```

Comment:
This is a duplicate of this cmdlet: https://github.com/Azure/azure-powershell/blob/master/src/Monitor/Monitor/Az.Monitor.psd1#L85

You may not duplicate existing cmdlet names

Reply:
Changes the cmdlet name to Get-AzAlertObjectHistory